src/foundry/client/pixi/perception/perception-manager.d.ts

```diff
@@ -1,107 +1,143 @@
-   * When a refresh occurs, the staged parameters are reset to these initial values.
-   */
-  static DEFAULTS: PerceptionManager.Options;
+    /** Update the configuration of vision sources */
+    refreshVisionSources: boolean;
 
-  /**
-   * The configured parameters for the next refresh.
-   */
-  params: PerceptionManager.Options;
+    /** Refresh the rendered appearance of vision */
+    refreshVision: boolean;
 
-  /**
-   * Cancel any pending perception refresh.
-   */
-  cancel(): void;
+    /** Re-initialize the entire ambient sound configuration */
+    initializeSounds: boolean;
 
-  /**
-   * Schedule a perception update with requested parameters.
-   * @param options - (default: `{}`)
-   */
-  schedule(options?: DeepPartial<PerceptionManager.Options>): void;
+    /** Refresh the audio state of ambient sounds */
+    refreshSounds: boolean;
 
-  /**
-   * Perform an immediate perception update.
-   * @param options - (default: `{}`)
-   */
-  update(options?: DeepPartial<PerceptionManager.Options>): void;
+    /** Apply a fade duration to sound refresh workflow */
+    soundFadeDuration: boolean;
 
-  /**
-   * A helper function to perform an immediate initialization plus incremental refresh.
-   */
-  initialize(): ReturnType<this["update"]>;
+    /** Refresh the visual appearance of tiles */
+    refreshTiles: boolean;
 
-  /**
-   * A helper function to perform an incremental refresh only.
-   */
-  refresh(): ReturnType<this["update"]>;
+    /** Refresh the contents of the PrimaryCanvasGroup mesh */
+    refreshPrimary: boolean;
+  }
 
   /**
-   * Set option flags which configure the next perception update
+   * A helper class which manages the refresh workflow for perception layers on the canvas.
+   * This controls the logic which batches multiple requested updates to minimize the amount of work required.
+   * A singleton instance is available as canvas#perception.
+   * @see Canvas#perception
    */
-  protected _set(options: DeepPartial<PerceptionManager.Options>): void;
+  class PerceptionManager extends RenderFlagsMixin(Object) {
+    static RENDER_FLAGS: {
+      initializeLighting: { propagate: ["refreshLighting", "refreshVision"] };
+      refreshLighting: { propagate: ["refreshLightSources"] };
+      refreshLightSources: Record<string, never>;
+      refreshVisionSources: Record<string, never>;
+      refreshPrimary: Record<string, never>;
+      initializeVision: {
+        propagate: ["refreshVision", "refreshTiles", "refreshLighting", "refreshLightSources", "refreshPrimary"];
+      };
+      refreshVision: { propagate: ["refreshVisionSources"] };
+      initializeSounds: { propagate: ["refreshSounds"] };
+      refreshSounds: {};
+      refreshTiles: { propagate: ["refreshLightSources", "refreshVisionSources"] };
+      soundFadeDuration: {};
+      identifyInteriorWalls: { propagate: ["initializeLighting", "initializeVision"] };
+      forceUpdateFog: { propagate: ["refreshVision"] };
+    };
```

Comment:
Please note that by using literal strings and arrays, it'd be impossible to override them. At least in a type safe way.

I'd put this in a `@defaultValue` and if I had to guess most of these would be `Array<keyof PerceptionManagerFlags>`. I honestly have never used these.